axelrod/fingerprint.py

```diff
@@ -240,34 +90,39 @@ def fingerprint(self, turns=50, repetitions=10, step=0.01, processes=None,
         turns : integer, optional
             The number of turns per match
         repetitions : integer, optional
-            The number of times the round robin should be repeated
+            The number of times the each match is repeated
         step : float, optional
+            0.0 < step <= 1.0
             The separation between each Point. Smaller steps will
             produce more Points that will be closer together.
         processes : integer, optional
             The number of processes to be used for parallel processing
+        filename :
+            Location of csv of tournament.interactions_dict
+        in_memory :
+            True=tournament.interactions_dict is stored only in memory
+            False=tournament.interactions_dict is stored only on file
         progress_bar : bool
-            Whether or not to create a progress bar which will be updated
+            Whether or not to show progress bars during steps of data creation.
 
         Returns
         ----------
-        self.data : dictionary
-            A dictionary where the keys are coordinates of the form (x, y) and
-            the values are the mean score for the corresponding interactions.
+        self.data : dictionary(Point, float)
+            The mean score for all matches against the probe corresponding to
+            each Point
         """
 
         if on_windows and (filename is None):  # pragma: no cover
             in_memory = True
         elif filename is None:
-            outputfile = NamedTemporaryFile(mode='w')
-            filename = outputfile.name
-
-        edges, tourn_players = self.construct_tournament_elements(
+            output_file = NamedTemporaryFile(mode='w')
```

Comment:
the linter complained that `outputfile` and `tourn_players` were typos, but `output_file` and `tournament_players` were ok. it seemed reasonable.